tools/RAiDER/delay.py

```diff
@@ -78,6 +78,7 @@ def tropo_delay(
         else:
             height_levels = wm_levels
 
+    zref = float(zref)
```

Comment:
This seems like this fix unrelated to PR that you are putting here. Please explain.

Reply:
a string can be entered instead of a float/int by user. catching here. given our timelines, it's easier to just include here